pkg/apis/pipeline/v1beta1/swagger.json

```diff
@@ -0,0 +1,2186 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Python SDK for Tekton Pipeline",
```

Comment:
Is this right? If this is python specific it should probably be in the experimental repo?

Reply:
Good catch! updated to "Tekton Pipeline",  we used the swagger file to generate Python SDK, yeah, there should be a common name.